### ISSUE TITLE
About multi-view evaluation

### ISSUE THREAD

**Issue by XiaobenLi00** [nkolot/ProHMR#32]
I noticed that in multi-view setup, gt 3d keypoints are repeated
`pred_keypoints_3d = output['pred_keypoints_3d'].detach()`
 `batch_size = pred_keypoints_3d.shape[0]`
`num_samples = pred_keypoints_3d.shape[1]`
`gt_keypoints_3d = batch['keypoints_3d'][:, :, :-1].unsqueeze(1).repeat(1, num_samples, 1, 1)`
So I wonder that which coordinate frame are the `gt_keypoints_3d` in ? In the data processing, I believe `gt_keypoints_3d` are in camera coordinate frame, so why are they repeated for different views?
